live-examples/html-examples/abbr.html

```diff
@@ -0,0 +1 @@
+<p>You can use <abbr title="Cascading Style Sheets">CSS</abbr> to style your <abbr title="HyperText Markup Language">HTML</abbr>.</p>
```

Comment:
I wonder, should we break this onto two lines to avoid horizontal scrolling?

Reply:
I thought it might be better for the editor to wrap lines, as I suggested in https://github.com/mdn/interactive-examples/issues/699. Since the editor is resizable, it's hard to know at what width we should wrap, if we have to do it manually.